scripts/kind-e2e/lib_subctl_gather_test.sh

```diff
@@ -90,3 +90,12 @@ function validate_resource_files() {
   done
 }
 
+function validate_broker_resources() {
+
+  ${DAPPER_SOURCE}/bin/subctl gather --kubeconfig ${KUBECONFIGS_DIR}/kind-config-${cluster} --dir $out_dir
+
+  validate_resource_files $SUBMARINER_BROKER_NS 'endpoints.submariner.io' 'Endpoint'
+  validate_resource_files $SUBMARINER_BROKER_NS 'clusters.submariner.io' 'Cluster'
+  validate_resource_files all 'serviceimports.multicluster.x-k8s.io' 'ServiceImport'
+  validate_resource_files all 'endpointslices.discovery.k8s.io' 'EndpointSlice'
```

Comment:
They're synced to the broker NS.

```suggestion
  validate_resource_files $SUBMARINER_BROKER_NS 'serviceimports.multicluster.x-k8s.io' 'ServiceImport'
  validate_resource_files $SUBMARINER_BROKER_NS 'endpointslices.discovery.k8s.io' 'EndpointSlice'
```